spec/features/enter_name_spec.rb

```diff
@@ -1,6 +1,8 @@
-feature 'Enter name page' do
-  scenario 'Allows player to enter their name' do
-    visit ('/enter-name')
-    expect(page).to have_content ('Enter your name')
+feature 'Enter names' do
+  scenario 'Player 1 submits their name' do
+    visit('/')
+    fill_in :player_1_name, with: 'CJ'
```

Comment:
I'm wondering whether the first part needs to be a string rather than symbol, that's why it may not be working. So 'player_1_name' .